core/common/src/main/java/alluxio/util/IdUtils.java

```diff
@@ -34,6 +34,7 @@
   public static final long INVALID_WORKER_ID = -1;
   public static final long INVALID_MOUNT_ID = -1;
   public static final long ROOT_MOUNT_ID = 1;
+  public static final long JOURNAL_MOUNT_ID = Long.MAX_VALUE - 10000;
```

Comment:
NIT: I think this only applies to UFS journal, so maybe rename this to`UFS_JOURNAL_MOUNT_ID`, and add a comment that the journal ufs is stored as a special mount in the ufs manager.

Reply:
Done